### PR TITLE
Update TypeScript to 2.7.2 version

### PR DESCRIPTION
**Pull request by benoitf**
With TypeScript 2.7.1 there was a problem
https://github.com/theia-ide/theia/issues/1239 fixed in https://github.com/Microsoft/TypeScript/issues/21478 

so upgrading to TypeScript 2.7.2

Change-Id: I1aa71c50873baa380cc03dc49d55475fe2f10a11
Signed-off-by: Florent BENOIT <fbenoit@redhat.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "ts-node": "^3.2.0",
     "tslint": "^5.7.0",
     "typedoc": "^0.8",
-    "typescript": "^2.7.1",
+    "typescript": "2.7.2",
```
Comment (akosyakov) [inline package.json]: please keep the caret, yarn.lock will lock the version anyway

Comment (benoitf) [reply package.json]: it was if ppl were not using yarn

File: yarn.lock [hunk 2]
```diff
@@ -7811,9 +7811,9 @@ typescript@2.4.1:
   version "2.4.1"
   resolved "https://registry.yarnpkg.com/typescript/-/typescript-2.4.1.tgz#c3ccb16ddaa0b2314de031e7e6fee89e5ba346bc"
 
-typescript@^2.7.1:
-  version "2.7.1"
-  resolved "https://registry.yarnpkg.com/typescript/-/typescript-2.7.1.tgz#bb3682c2c791ac90e7c6210b26478a8da085c359"
+typescript@2.7.2:
```
Comment (kittaakos) [inline yarn.lock]: I am not sure, so I am instead just asking; should we have the caret here too, just like before?

**Comment by benoitf** [comment]
@akosyakov review is ok ?

**Pull request merged**